Dockerfile

```diff
@@ -160,24 +160,6 @@ COPY . /work
 WORKDIR /work
 RUN ./pkg/rpm/build.sh
 
-FROM centos:8 AS centos8-build
```

Comment:
We do need a RHEL 8 build. The right approach here is to replace CentOS 8 by Rocky Linux 8 (i.e., use `rockylinux:8` here).